app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -18,41 +18,49 @@ def get_openshift_images(inventory)
       end
 
       def get_builds(inventory)
-        process_collection(inventory["build_config"], :container_builds) { |n| parse_build(n) }
+        builds = map_entity("build_config")
+        process_collection(inventory["build_config"], builds) { |n| parse_build(n) }
 
-        @data[:container_builds].each do |ns|
-          @data_index.store_path(:container_builds, :by_name, ns[:name], ns)
+        @data[builds].each do |ns|
+          @data_index.store_path(builds, :by_name, ns[:name], ns)
         end
       end
 
       def get_build_pods(inventory)
-        process_collection(inventory["build"], :container_build_pods) { |n| parse_build_pod(n) }
+        build_pods = map_entity("build")
+        process_collection(inventory["build"], build_pods) { |n| parse_build_pod(n) }
 
-        @data[:container_build_pods].each do |ns|
-          @data_index.store_path(:container_build_pods, :by_name, ns[:name], ns)
+        @data[build_pods].each do |ns|
+          @data_index.store_path(build_pods, :by_name, ns[:name], ns)
         end
       end
 
       def get_routes(inventory)
-        process_collection(inventory["route"], :container_routes) { |n| parse_route(n) }
+        process_collection(inventory["route"], map_entity("route")) { |n| parse_route(n) }
       end
 
       def get_projects(inventory)
+        projects = map_entity("project")
```

Comment:
I don't know, maybe it's worth to:
```
data_index = map_entity("project")
```
?

That would bring us closer to extracting those methods (which we are not doing now)